styles.go

```diff
@@ -1032,6 +1032,65 @@ func (f *File) NewStyle(style *Style) (int, error) {
 	return setCellXfs(s, fontID, numFmtID, fillID, borderID, applyAlignment, applyProtection, alignment, protection)
 }
 
+// Value extracts string data type text from a attribute value.
```

Comment:
I suggest that move these types of functions implemented to the `lib.go` source file.

Reply:
Okay. I have moved these to the `lib.go` source file.